sdk/helper/useragent/useragent.go

```diff
@@ -31,6 +31,15 @@ var (
 // Given comments will be appended to the semicolon-delimited comment section.
 //
 // e.g. Vault/0.10.4 (+https://www.vaultproject.io/; go1.10.1; comment-0; comment-1)
+//
+// Deprecated: use PluginString instead.
```

Comment:
I vote for we rip the band-aid off and delete this function, the variables above, and the reference above to `sdk/version`, and bump the major version of `sdk`.

Reply:
Or just replace it with `PluginString()` below, and then we don't have to bump the `sdk` version.

Reply:
+1 to replacing with PluginString

Reply:
How do you mean replace it?  They have different arguments.

Reply:
Ah, I missed that. It looks like it will be non-trivial to replace some usages (like in agent). When I first looked I thought this function was mostly unused in our code base. Nevermind.

Reply:
What difficulty do you see with modifying agent?

Reply:
At a glance, I don't see a `logical.PluginEnvironment` in the agent to pass to `PluginString()`.

Reply:
We spoke on slack, Chris's agent comment was re the idea of removing String().  But Since agent lives within the vault package, it now uses the internal useragent helper rather than sdk's, so that's not actually an impediment.

I'm starting to come around to the idea of removing useragent.String from the sdk.  I don't think we actually need to bump the major since we're pre-1.0.  Or do you mean going from v0.6 to v0.7?

Reply:
Bumping from 0.6 to 0.7 is what I meant.